config/config.go

```diff
@@ -15,74 +15,77 @@ type (
 	// Config contain backend Configuration
 	Config struct {
 		// --- General Configuration ---
-		Port int    `json:"port"` // Default: 8080
-		Env  string `json:"env"`  // Default: production
+		Port int    // Default: 8080
+		Env  string // Default: production
 
 		// --- Cache Configuration ---
 		// UpstreamCacheExpiration is used to respond before executing the request. Avoid overloading services.
-		UpstreamCacheExpiration int `json:"upstreamCacheExpiration"`
+		UpstreamCacheExpiration int
 		// DownstreamCacheExpiration is used to respond after executing the request in case of timeout error.
-		DownstreamCacheExpiration int `json:"downstreamCacheExpiration"`
+		DownstreamCacheExpiration int
 
 		// Monitorable Config
-		Monitorable Monitorable `json:"monitorable"`
+		Monitorable Monitorable
 	}
 
 	Monitorable struct {
-		Ping        map[string]*Ping        `json:"ping"`
-		Port        map[string]*Port        `json:"port"`
-		HTTP        map[string]*HTTP        `json:"http"`
-		Pingdom     map[string]*Pingdom     `json:"pingdom"`
-		Github      map[string]*Github      `json:"github"`
-		TravisCI    map[string]*TravisCI    `json:"travisCI"`
-		Jenkins     map[string]*Jenkins     `json:"jenkins"`
-		AzureDevOps map[string]*AzureDevOps `json:"azureDevOps"`
+		Ping        map[string]*Ping
+		Port        map[string]*Port
+		HTTP        map[string]*HTTP
+		Pingdom     map[string]*Pingdom
+		TravisCI    map[string]*TravisCI
+		Jenkins     map[string]*Jenkins
+		AzureDevOps map[string]*AzureDevOps
+		Github      map[string]*Github
```

Comment:
Alphabetical ordered?

Reply:
Hum, so we need to change in everywere ?